zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/tenant/TenantQueryControllerTest.java

```diff
@@ -562,8 +564,19 @@ void shouldListMembersOfTypeClient() {
         .json(
             // language=json
             """
-                {"items":[{"clientId":"client1"},{"clientId":"client2"},{"clientId":"client3"}],"page":{"totalItems":3}}
-            """);
+                {
```

Comment:
I like that you made those more readable :D

Reply:
My eyes start bleeding when I see such things, I HAD to adjust this 😭